src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -651,15 +651,14 @@ public InputStream fileCardImage(@PathParam("fileId") Long fileId, @Context UriI
                         || "application/zipped-shapefile".equalsIgnoreCase(df.getContentType())) {
 
                     thumbnailDataAccess = ImageThumbConverter.getImageThumbnailAsInputStream(dataAccess, 48);
+                    if (thumbnailDataAccess != null && thumbnailDataAccess.getInputStream() != null) {
```

Comment:
Not sure if it applies here or not, but in past leak hunting removed several `if (foo.getInputStream() != null)` type statements, where the anonymous open statements appeared to be correlated with accumulating open file descriptors.

Reply:
@pameyer - I think it should be OK in this case since the StorageIO class caches the InputStream - repeated calls to it don't create more streams as it would if you were directly opening a file stream, etc.
FWIW - In checking on this, I did find that that the code retrieved the channel, created from a stream, in order to create a new stream, rather than just retrieving the stream directly. Added a fix as a new commit.